tests/svg_transform_test.py

```diff
@@ -116,14 +116,39 @@ def test_scale_0_is_degenerate(self):
         assert Affine2D.identity().scale(1, 0).is_degenerate()
         assert Affine2D.identity().scale(0, 0).is_degenerate()
 
-    def test_inverse(self):
-        t = Affine2D.identity().translate(2, 3).scale(4, 5)
+    @pytest.mark.parametrize(
+        "transform, inverse",
+        [
+            (
+                Affine2D.identity().translate(2, 3).scale(4, 5),
+                Affine2D(a=0.25, b=0.0, c=0.0, d=0.2, e=-0.5, f=-0.6),
+            ),
+            (
+                Affine2D(0, -2.875, 2.875, 0, 2.39, 250.81),
+                Affine2D(0.0, 0.348, -0.348, 0.0, 87.238, -0.831),
+            ),
+        ],
+    )
+    def test_inverse(self, transform, inverse):
+        assert transform.inverse().almost_equals(inverse, 1e-3)
+        assert (transform @ transform.inverse()).almost_equals(Affine2D.identity())
+        assert (transform.inverse() @ transform).almost_equals(Affine2D.identity())
+
+    @pytest.mark.parametrize(
+        "transform",
+        [
+            Affine2D.identity().translate(2, 3).scale(4, 5),
+            Affine2D(0, -2.875, 2.875, 0, 2.39, 250.81),
+        ],
+    )
+    def test_inverse_roundtrip(self, transform):
         p0 = Point(12, 34)
-        p1 = t.map_point(p0)
-        it = t.inverse()
+        p1 = transform.map_point(p0)
+        it = transform.inverse()
         p2 = it.map_point(p1)
         assert p2 == p0
```

Comment:
Perhaps also assert p1 != p0 unless transform is identity. On which note it's probably worth testing that identity doesn't do anything if we don't already, perhaps as a separate test.